ironfish-rust/src/merkle_note.rs

```diff
@@ -138,7 +141,7 @@ impl MerkleNote {
             value_commitment: value_commitment.commitment().into(),
             note_commitment: note.commitment_point(),
             ephemeral_public_key: (*public_key),
-            encrypted_note,
+            encrypted_note: encrypted_note.unwrap(),
```

Comment:
We could probably avoid the unwrapping in this one as well, if you want to do that in a follow-up